src/platform/zc706/config.sh

```diff
@@ -0,0 +1,111 @@
+#! /bin/bash --
+
+# This configure file is used by src/runtime_src/tools/scripts/peta_build.sh
+# You could override below functions in this configure script
+#	config_peta     -- configure petalinux project.
+#	config_kernel   -- configure linux kernel.
+#	config_rootfs   -- configure rootfs.
+#	config_dts      -- configure system user device tree.
+#	install_recipes -- install recipes to petalinux.
+#	rootfs_menu     -- update rootfsconfig to add package.
+#	pre_build_hook  -- just before petalinux-build.
+#	post_build_hook -- just after petalinux-build.
+
+THIS_CONFIG_SCRIPT_DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null 2>&1 && pwd )"
+
+# PetaLinux template. Default is zynqMP
+TEMPLATE=zynq
+
+# peta_build.sh will set XRT_REPO_DIR pointed to the XRT repository.
+# This would be helpful when you want to use some resources in XRT repo
+
+# The first argument is the petalinux configure file
+#  config_peta <petalinux_project>/project-pec/configs/config
+#
+#config_peta()
+#{
+#	PETA_CONFIG_FILE=$1
+#	echo "CONFIG_YOCTO_ENABLE_DEBUG_TWEAKS=y" >> $PETA_CONFIG_FILE
+#}
+
+# The first argument is the linux kernel configure file
+#  config_kernel recipes-kernel/linux/linux-xlnx/user.cfg
+config_kernel()
+{
+	KERN_CONFIG_FILE=$1
+	# *** Enable or disable Linux kernel features as you need ***
+	# AR# 69143 -- To avoid PetaLinux hang when JTAG connected.
+	echo '# CONFIG_CPU_IDLE is not set' >> $KERN_CONFIG_FILE
+	echo 'CONFIG_XILINX_INTC=y' >> $KERN_CONFIG_FILE
+}
+
+# The first argument is the rootfs configure file
+#  config_rootfs project-spec/configs/rootfs_config
+#config_rootfs()
+#{
+#	ROOTFS_CONFIG_FILE=$1
+#	echo 'CONFIG_xrt=y'                 >> $ROOTFS_CONFIG_FILE
+#	echo 'CONFIG_mnt-sd=y'              >> $ROOTFS_CONFIG_FILE
+#	echo 'CONFIG_xrt-dev=y'             >> $ROOTFS_CONFIG_FILE
+#	echo 'CONFIG_zocl=y'                >> $ROOTFS_CONFIG_FILE
+#	echo 'CONFIG_opencl-headers-dev=y'  >> $ROOTFS_CONFIG_FILE
+#	echo 'CONFIG_opencl-clhpp-dev=y'    >> $ROOTFS_CONFIG_FILE
```

Comment:
Should we add OpenCV to the rootfs?

Reply:
I am waiting for the change done in mpsoc platforms get working with build team. then Will add the same for all zynq7000 platforms.

Reply:
Sure.